x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts

```diff
@@ -18,7 +18,11 @@ const configSchema = schema.object({
     schema.maybe(schema.string())
   ),
   eventTypesAllowlist: schema.arrayOf(schema.string(), {
-    defaultValue: ['Loaded Kibana'],
+    defaultValue: [
+      'Loaded Kibana', // Sent once per page refresh (potentially, once per session)
+      'Hosts View Query Submitted', // Worst-case scenario 1 every 2 seconds
+      'Host Entry Clicked', // Worst-case scenario once per second - AT RISK,
```

Comment:
@elasticelvin and @shahinakmal, bear in mind these new _Clicked_ events might throttle the collection of FullStory if the user clicks and submits queries 10 times in a second or 30 in a minute.

To reduce the risks, I'd recommend tracking clicks as discussed in https://github.com/elastic/kibana/pull/149497#discussion_r1087593410.

However, to unblock this PR so it can make it through our FF, I'll approve it. But we may want a follow-up PR to rework this, and a proper discussion about how to track these sort of events (I bet FS' automatic collection can easily provide the same info as these 2 custom events without any limit).